ui/packages/.packages_release_version

```diff
@@ -1,3 +1,3 @@
 # This file contains a version number which will be used to publish ui packages.
 # To trigger a new publish, simply increase this version number.
```

Comment:
To trigger a new publish, increase this version number, as well as bump all package versions by running `yarn run bump:packages`.

Reply:
good advice, will add in the next PR.